pyroma/ratings.py

```diff
@@ -346,14 +347,27 @@ def test(self, data):
                 # license classifier exist
                 licenses.add(classifier)
 
-        if not license and not licenses:
+        if not license and not license_expression and not licenses:
             self._message = "Your package does neither have a license field nor any license classifiers."
             return False
 
-        if license in CODE_LICENSES:
-            if not CODE_LICENSES[license].intersection(licenses):
-                self._message = f"The license '{license}' specified is not listed in your classifiers."
-                return False
+        if license and license_expression:
+            self._message = "You do not need to specify both a License and a Licence-Expression."
+            return False
```

Comment:
As a matter of fact, not only is this unnecessary but also deprecated, is rejected by PyPI on upload, and impossible to do with modern pyproject.toml-based metadata. I suggest a stronger statement here, e.g. "Specifying both a License and a License-Expression is ambiguous, deprecated, and may be rejected by package indices."

Reply:
Oh, it gets rejected? Yeah, I'll change that.

Reply:
Yeah, that's what we ended up specifying in the PEP (though I haven't personally tested it on PyPI yet)